### DISCUSSION TITLE
Second value during initial adjustment

### DISCUSSION THREAD

**Discussion by PitToad** [rsjaffe/MIDI2LR#679] [Problems]
LR 11.4.1, MidiLR 5.4.0.0
Sony RAW Files

When editing a new RAW/ARW file, the adjustments don't alway start at zero, and I have to adjust up or down to zero for Lightroom to then reflect adjustments.

Using Midifighter Twister or Arturia Beatstep have same effect - Is there something I've not ticked?

Adjustment doesn't always start at zero, even on newly imported, unedited pics - the first value is the Clarity value last made on a previous pic and the '0' is the Clairty adjustment for this pic
![cla2](https://user-images.githubusercontent.com/111119318/184327106-f2e33cf4-3c9e-44df-899a-9b4f0b218b25.jpg)

Adjusting Carity + has no effect on Clarity slider or value
![cla3](https://user-images.githubusercontent.com/111119318/184327163-db23ef3f-834b-4905-b712-6553da11d4cd.jpg)

Once encoder goes past zero (Now into Clarity negative values, so past zero from positive values) , adjustments are visible on Lightroom
![cla5](https://user-images.githubusercontent.com/111119318/184327192-55984926-e0e4-4b36-bab6-fe5c9acd47fb.jpg)

**Comment by rsjaffe** [comment]
Check your [pickup mode](https://github.com/rsjaffe/MIDI2LR/wiki/Application-Settings-Dialog).

**Comment by PitToad** [comment]
Sorry - it's not taking the adjustment to zero - it's getting the 'live' adjustments to match the second number before anything happens to the image
I've uploaded a video of what happens here; https://drive.google.com/drive/folders/1dl8TKVP5WycQ8Klkhl4XB0pT1ytAu0ny?usp=sharing

If the display shows Clarity 0 and second set of numbers is 54 (for instance) the adjustments don't show on screen until 54 is reached with the clarity adjustment.
Most of the time the second number is zero, as I'm working on unedited images, but I've just noticed that if the second number isn't zero, then the adjustment needs to match that second number before adjustments show

Pickup mode is already enabled - if switched off, the trailing number disappears, but adjustments on the next pic start from the setting previous.
So, if pic 1 is Clarity 5, Pic 2 Clarity adjustments start at 5 when the dial is rotated

![CLA4](https://user-images.githubusercontent.com/111119318/184507849-a5aae4b9-f45a-4080-aa2c-72d0c68b8759.jpg)

![clA7](https://user-images.githubusercontent.com/111119318/184507856-1a9bb72e-2a81-4944-a8fb-4a1b921c9fc3.jpg)

Midi2lr registers adjustments just above the Rescan button, but adjustments don't show in Lightroom until control dial adjustment matches second set of numbers
![clA6](https://user-images.githubusercontent.com/111119318/184507862-3a6c13f6-1aad-48b5-8afd-8eca36b8b92c.jpg)


Clarity and Brightness dial settings in Arturia setup;
![cla10](https://user-images.githubusercontent.com/111119318/184507871-1df65a61-59aa-45bf-8a9e-5c8dee9e4a2b.jpg)

![cla11](https://user-images.githubusercontent.com/111119318/184507879-14bceb2c-0355-47ef-897c-d1f259eb25cb.jpg)

**Comment by PitToad** [comment]
Kinda got it working;
In Arturia software, changed 'Option' on each dial to 'Relative # 1' (had it on 'Absolute' originally see lower two pics in above post)
In Midi2LR changed CC Dialog to 'Binary Offset' 

Compared to Midifighter Twister though, dials don't change values in whole numbers on-screen adjustment shows increase/decrease steps around 0.15 - 0.16 instead of 1 but that doesn't affect usability at all

![beats1](https://user-images.githubusercontent.com/111119318/185302247-d327cb7e-693f-479a-b189-d18965eed449.jpg)

![beats3](https://user-images.githubusercontent.com/111119318/185302289-af12a3b6-67f4-4e89-be6a-ea21dd34e0fb.jpg)


![exp1](https://user-images.githubusercontent.com/111119318/185305265-9d1a61b9-4559-40ab-baba-2e0bfaa910c1.jpg)

![exp2](https://user-images.githubusercontent.com/111119318/185305287-3202b634-7acb-4835-a83d-3a3f228265c4.jpg)

**Comment by jimvetter** [comment]
I'm having the same problem. It just started for me. Perhaps related to my using both and XTouch and a BCR2000 at the same time.

Has no one responded to you?

I'm curious what the Arturia software is. Can you use it to program the BCR2000?

Thanks!

Jim

**Comment by PitToad** [comment]
Hi Jim

I used an X Touch Compact ok, and a BCF2000 before that without any problem
My Midifighter Twister has just started showing a trailing zero on some adjustments now too, so no idea if problem is Midi2LR or Lightroom itself - I've not updated Midi2LR for a few weeks/months but the Midifighter issue is new, certainly since my last post above.

The Arturia software detects an Arturia deck is plugged in, so that wouldn't work on your BCR, but there was some software I had to install to get it working initially. If your BCR has worked before now though, it's unlikely to be that.

On my setup, I can have the Arturia Beatstep, Midi Twister and Midi 3d all plugged in and working, I just had to change the Midi channel on each one so they weren't all using the same channel.
There's no need for me to have them all plugged in, but it does mean if I forget to turn one off it doesn't conflict with the others.

My initial issues seemed to have been caused by controller setup rather than Midi2LR itself,  so I resolved it (until the latest zero issues) by swapping settings on the Arturia software.
